software/ybfeed.yml

```diff
@@ -0,0 +1,12 @@
+name: "ybFeed"
+website_url: "https://github.com/ybizeul/ybFeed"
+source_code_url: "https://github.com/ybizeul/ybFeed"
+description: "ybFeed is a personal micro feed where you can post snippets of text or images"
```

Comment:
```suggestion
description: "ybFeed is a personal micro feed where you can post snippets of text or images."
```

Reply:
Will do. It's a bit confusing as what should go in that field. 
At the end of the day, this is a binary that can run standalone or in a docker container. If compiling it yourself or developing then yes it relies on Go and NodeJS